test/integration/fortios.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I don't think you need this file as `ansible-test` will create one, also it could get out of sync with the actual tests.


ansible-test integration -vvve fortios.*

The `-e` means `explain`, notice that a Playbook is generated on the fly